gradle/dist.gradle

```diff
@@ -0,0 +1,89 @@
+import java.nio.file.Paths
+
+import static groovy.io.FileType.FILES
+
+def licenses = [
+        [name: 'avalon-framework', title: 'Avalon Framework', license: ['apache2.txt']],
+        [name: 'batik', title: 'Batik', license: ['apache2.txt']],
+        [name: 'xmlgraphics-commons', title: 'Apache XML Graphics Commons', license: ['apache2.txt']],
+        [name: 'commons-io', title: 'Commons IO', license: ['apache2.txt']],
+        [name: 'xercesImpl', title: 'Xerces2 Java', license: ['apache2.txt']],
+        [name: 'xml-apis', title: 'XML Standard API', license: ['apache2.txt']],
+        [name: 'xml-resolver', title: 'Resolver', license: ['apache2.txt']],
+        [name: 'Saxon-HE', title: 'Saxon-HE', license: ['mpl2.txt', 'CERN.txt', 'JAMESCLARK.txt', 'THAI.txt', 'UNICODE.txt']],
+        [name: 'icu4j', title: 'ICU4J', license: ['icu.txt']],
+        [name: 'ant', title: 'Ant', license: ['apache2.txt']],
+        [name: 'guava', title: 'Guava', license: ['apache2.txt']],
+        [name: 'slf4j', title: 'SLF4J', license: ['mit-slf4j.txt']],
+        [name: 'jcl-over-slf4j', title: 'SLF4J', license: ['mit-slf4j.txt']],
+        [name: 'logback', title: 'Logback', license: ['epl-1.0.txt']],
+        [name: 'jackson', title: 'Jackson', license: ['apache2.txt']],
+        [name: 'jing', title: 'Jing', license: ['bsd-jing.txt']],
+        [name: 'gradle-wrapper'],
+        [name: 'fo'],
+        [name: 'eclipsehelp'],
+        [name: 'eclipsehelp'],
+        [name: 'htmlhelp'],
+        [name: 'htmlhelp'],
+        [name: 'xep'],
+        [name: 'axf'],
+        [name: 'dost'],
+        [name: 'error_prone_annotations'],
+        [name: null, title: 'DITA 1.1 DTDs and Schemas', license: ['oasis-dita11.txt']],
+        [name: null, title: 'DITA 1.2 DTDs and Schemas', license: ['oasis-dita12.txt']]
+]
+
+task generateNotices() {
+    doLast {
+        file(distTempDir).traverse(type: FILES, nameFilter: ~/.*\.jar$/) {
+            def name = it.getName()
+            def key = licenses.find { it.name != null && name.startsWith(it.name) }
+            if (key == null) {
+                throw new GradleScriptException("License for ${name} not found", new RuntimeException());
+            }
+        }
+
+        file("${distTempDir}/NOTICES.txt").withWriter('UTF-8') { out ->
+            out.writeLine("""DITA Open Toolkit: Third Party Notices
+
+This Notices file contains certain notices and important information that the
```

Comment:
This header comes from the old NOTICES.txt, probably originally from IBM. Ping @robander

Reply:
Probably correct, though that would have gone back to release DITA-OT 1.0 and I didn't get involved in this part of the project until right _after_ that, so I can't confirm (Original project lead Don and any lawyers involved at that time have all retired by now). Were you hoping to remove the text or just looking for the source?